go/obscurocommon/l1_transaction.go

```diff
@@ -0,0 +1,40 @@
+package obscurocommon
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// L1Transaction represents how obscuro interprets Transactions that happen on the Layer 1
+// These transactions can have different implementations
+type L1Transaction interface {
+	Bytes() []byte
```

Comment:
Is this "Bytes()" function used anywhwere?

Reply:
It was! Not anymore, removed 👍